modules/oe_content_call_proposals/oe_content_call_proposals.module

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * @file
+ * Primary module hooks for OpenEuropa Call For Proposals content module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\oe_content_call_proposals\CallForProposalsNodeWrapperInterface;
+
+/**
+ * Implements hook_form_BASE_FORM_ID_alter for the node form.
+ */
+function oe_content_call_proposals_form_node_form_alter(array &$form, FormStateInterface $form_state) {
+  /** @var \Drupal\node\NodeInterface $node */
+  $node = $form_state->getBuildInfo()['callback_object']->getEntity();
+  if ($node->bundle() !== 'oe_call_proposals') {
+    return;
+  }
+
+  $form['oe_call_proposals_deadline']['widget']['#states'] = [
+    'invisible' => [
```

Comment:
If "Deadline model" is "Permanent" this field is not displayed in the backend, for some reason, this does not work in my test

Reply:
Can you please test again? The `Deadline model` is a radio buttons set now and it should work.

Reply:
There is a notice `Notice: Undefined index: #type in Drupal\Core\Form\FormHelper::processStates() (line 208 of core/lib/Drupal/Core/Form/FormHelper.php).` with current implementation on the `/node/add/oe_call_proposals` page.

But it works good: `$form['oe_call_proposals_deadline']['#states'] = [`.

Reply:
I fixed the problem.